fast64_internal/sm64/sm64_anim.py

```diff
@@ -697,7 +696,7 @@ def readValueIndex(romfile, startAddress):
 
     # multiply 2 because value is the index in array of shorts (???)
     startOffset = int.from_bytes(romfile.read(2), "big") * 2
-    #print(str(hex(startAddress)) + ": " + str(numFrames) + " " + str(startOffset))
+    # print(str(hex(startAddress)) + ": " + str(numFrames) + " " + str(startOffset))
```

Comment:
is this comment useful?

Reply:
guess not, we can cleanup these later

or, logging would be cool and this would go to debug/trace levels

since black format PRs are large and automatic I'd rather not pull small manual changes in